spec/tests/helpers/perf_run_helper_spec.rb

```diff
@@ -746,31 +746,72 @@ class PerfRunHelperClass
+        expect(subject.validate_baseline_data(data)).to eq(true)
       end
     end
-    context "when any delta is > MAX_BASELINE_VARIANCE" do
+    context "when any variance is > MAX_BASELINE_VARIANCE lower" do
       it "returns false" do
-        expect(subject.validate_baseline_data("fail_thing"  => [1, 2],
-                                              "pass_thing2" => [100, 102])).to eq(false)
+        baselines = [1.00, 100]
+        data = { fail_lower: [baselines[0],
+                              baselines[0] - baselines[0] * PerfRunHelper::MAX_BASELINE_VARIANCE * 1.1],
+                 pass_higher: [baselines[1],
+                               baselines[1] + baselines[1] * PerfRunHelper::MAX_BASELINE_VARIANCE * 0.9] }
+        expect(subject.validate_baseline_data(data)).to eq(false)
       end
     end
-    context "when MAX_BASELINE_VARIANCE < 'orchestration_service memory delta' > MAX_BASELINE_VARIANCE_ORCH_REL_MEM" do
+    context "when any variance is > MAX_BASELINE_VARIANCE higher" do
+      it "returns false" do
+        baselines = [1.00, 100]
+        data = { fail_higher: [baselines[0],
+                               baselines[0] + baselines[0] * PerfRunHelper::MAX_BASELINE_VARIANCE * 1.1],
+                 pass_higher: [baselines[1],
+                               baselines[1] + baselines[1] * PerfRunHelper::MAX_BASELINE_VARIANCE * 0.9] }
+        expect(subject.validate_baseline_data(data)).to eq(false)
+      end
+    end
+    context "when 'orchestration_service memory variance' between max and max override" do
+      it "returns true" do
+        between_variation = (PerfRunHelper::MAX_BASELINE_VARIANCE +
+                             PerfRunHelper::MAX_VARIANCE_OVERRIDE_PROCESS_ORCHESTRATION_SERVICES_RELEASE_AVG_MEM
+                            ) / 2
+        baselines = [1.00, 100]
+        data = { pass_lower: [baselines[0],
+                              baselines[0] - baselines[0] * PerfRunHelper::MAX_BASELINE_VARIANCE * 0.9],
+                 process_orchestration_services_release_avg_mem: [baselines[1],
+                                                                  baselines[1] + baselines[1] * between_variation] }
+        expect(subject.validate_baseline_data(data)).to eq(true)
+      end
+    end
+    context "when 'orchestration_service memory variance' greater than max override AND worse" do
+      it "returns false" do
+        baselines = [1.00, 100]
+        data = { pass_lower: [baselines[0],
+                              baselines[0] - baselines[0] * PerfRunHelper::MAX_BASELINE_VARIANCE * 0.9],
+                 process_orchestration_services_release_avg_mem: [baselines[1],
+                                                                  baselines[1] + baselines[1] * PerfRunHelper::MAX_VARIANCE_OVERRIDE_PROCESS_ORCHESTRATION_SERVICES_RELEASE_AVG_MEM * 1.1] } # rubocop:disable  Metrics/LineLength
```

Comment:
Since `PerfRunHelper::MAX_VARIANCE_OVERRIDE_PROCESS_ORCHESTRATION_SERVICES_RELEASE_AVG_MEM` is being used twice and requires disabling the LineLength cop would it make sense to assign this to a shorter-named variable here to improve readability and satisfy the cop?

Reply:
:-1: that seems like the tail wagging the dog to me.  Such a variable would obfuscate the constant.